.kokoro/build.sh

```diff
@@ -58,7 +61,7 @@ mvn install -B -V \
   -T 1C
 
 # prepend Kokoro root directory onto GOOGLE_APPLICATION_CREDENTIALS path
-if [[ ! -z "${GOOGLE_APPLICATION_CREDENTIALS}" ]]; then
+if [[ ! -z "${GOOGLE_APPLICATION_CREDENTIALS}" && "${GOOGLE_APPLICATION_CREDENTIALS}" != /* ]]; then
```

Comment:
I don't know what this is trying to do?

Reply:
When running my test locally the credentials file isn't in the kokoro directory, and I pass a fully qualified path as the env var. This additional condition ensures the value for `GOOGLE_APPLICATION_CREDIENTIALS` doesn't have a leading `/` before it prepends the kokoro dir to it.

Reply:
Ok, let's add to the comment above something like "if set and is a relative path"

Reply:
Done